bounties_api/analytics/management/commands/timeline_generator.py

```diff
@@ -233,12 +241,12 @@ def generate_timeline(time_frame, schema):
 
 class Command(BaseCommand):
     def handle(self, *args, **options):
-
+        BountiesTimeline.objects.all().delete()
         schemas_query = BountyState.objects.distinct('bounty__schemaName')
```

Comment:
Another issue, and this was already an issue, is the end date is based on the latest bounty. It should be based on the current UTC time. Otherwise, this means a date has no stamp or record until a bounty action happens in that date. Makes sense?  This is an issue.